Trimble.Modus.Components/Controls/TabBar/TMTabbedPage.cs

```diff
@@ -0,0 +1,347 @@
+﻿using System.Collections;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using Trimble.Modus.Components.Constant;
+using Trimble.Modus.Components.Helpers;
+
+namespace Trimble.Modus.Components;
+
+[ContentProperty(nameof(TabItems))]
+public partial class TMTabbedPage : ContentPage
+{
+    #region Private Fields
+    private Grid mainContainer;
+    private Grid tabStripContainer;
+    private CarouselView contentContainer;
+    private List<double> contentWidthCollection;
+    private ObservableCollection<TabViewItem>? contentTabItems;
+    #endregion
+    #region Public Fields
+    public ObservableCollection<TabViewItem> TabItems { get; set; } = new();
+    public delegate void TabSelectionChangedEventHandler(object? sender, TabSelectionChangedEventArgs e);
+    public event TabSelectionChangedEventHandler? SelectionChanged;
+    #endregion
+    #region Protected Fields
+    protected Grid _tabBarView = null;
```

Comment:
Not used

Reply:
protected Grid _tabBarView = null;
    protected List<View> cells;
    protected List<View> selectedCells;

Reply:
Removed